src/style.css

```
@@ -0,0 +1,73 @@
+body {
+  display: flex;
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 100px;
+  margin: 0;
+  flex-direction: row;
+  flex-wrap: wrap;
+}
+
+.block {
+  color: white;
+  display: flex;
+  height: 300px;
+  flex-direction: row;
+  justify-content: center;
+  align-items: center;
+  min-width: 300px;
+  flex-basis: 26%;
+  flex-grow: 1;
+}
+
+@media (min-width: 600px)  and (max-width:899px) {
```

You can simplify this:
```suggestion
@media (min-width: 600px) {
```